pennylane/devices/qubit/adjoint_jacobian.md

```diff
@@ -0,0 +1,158 @@
+# The adjoint jacobian algorithm
+
+## Full jacobian with expectation values
+
+For a statevector,
+
+$$
+| \Psi \rangle = U_n(x_n) \dots U_i (x_i) \dots U_0(x_0) |0\rangle
+$$
+
+we have the derivative of an expectation value of an observable $M$,
+
+$$
+\frac{\partial}{\partial x_i} \langle \Psi | M | \Psi \rangle
+$$
+
+$$
+= \langle 0| \dots \frac{\partial U_i^{\dagger}}{\partial x_i} \dots M \dots
+U_i \dots |0\rangle +
+\langle 0 | \dots U_i^{\dagger} \dots M \dots \frac{\partial U_i}{\partial x_i} \dots |0 \rangle
+$$
+
+$$
+= 2 \text{Re} \big( \langle \Psi | M U_n \dots \frac{\partial U_i}{\partial x_i}  \dots U_0 |0\rangle \big)
+$$
+
+We can then make the definitions:
+
+$$
+\langle b_i|  = 2 \langle \Psi | M U_n \dots U_{i+1}
+$$
+
+$$
+|k_i \rangle  = U_{i-1} \dots U_0 |0\rangle
+$$
+
+$$
+| \tilde{k}_i \rangle = \frac{\partial U_i}{\partial x_i} | k_i \rangle
+$$
+
+which allows us to expressed the expectation value derivative as:
+
+$$
+\text{Re}\big( \langle b_i | \tilde{k}_i \rangle \big)
+$$
+
+We can then also define $|k_i\rangle$ and $|b_i\rangle$ iteratively as:
+
+$$
+|b_{i-1} \rangle = U^{\dagger}_i |b_i\rangle \qquad |k_{i-1}\rangle = U^{\dagger}_{i-1} |k_i \rangle
```

Comment:
found the issue by trying things out in the [browser editor for this file](https://github.com/PennyLaneAI/pennylane/edit/adjoint-diff-state/pennylane/devices/qubit/adjoint_jacobian.md)
```suggestion
|b_{i-1} \rangle = U^{\dagger}_i |b_i\rangle \qquad |k\_{i-1}\rangle = U^{\dagger}\_{i-1} |k_i \rangle
```

Reply:
So weird. Any idea why? You think it would also complain about `|b_{i-1}`

Reply:
it's if there's a curly brace _before_ the underscore: https://github.com/orgs/community/discussions/65772